Cargo.toml

```diff
@@ -31,6 +31,7 @@ serde_json = { version = "1.0"}
 serde = { version = "1.0", features = ["derive"] }
 zombienet-sdk = { git = "https://github.com/r0gue-io/zombienet-sdk", branch = "pop", version = "0.1.0-alpha.1"}
 zombienet-support = { git = "https://github.com/r0gue-io/zombienet-sdk", branch = "pop", version = "0.1.0-alpha.1" }
+git2_credentials = "0.13.0"
```

Comment:
IIRC, git2 itself has methods that expose functionality to SSH clone repositories.

Reply:
@AlexD10S thoughts?

Reply:
Is `git2_credentials` needed or could we use this: https://docs.rs/git2/latest/git2/build/struct.RepoBuilder.html#example